### DISCUSSION TITLE
bogus PTR records in Deadwood

### DISCUSSION THREAD

**Discussion by rabsym** [samboy/MaraDNS#126] [Q&A] [answered]
Hello all,

Would like to ask if there is anyway to tell Deadwood use some bogus PTR records as IP4 or IP6 config keywords does...

ip4["maradns.foo."] = "10.10.10.10"
ip6["maradns.foo."] = "20010db84d617261444e530000001234"

maybe something like the following or perhaps any other tip:

ptr["192.168.1.100."] = "somehost.domain"

Thanks in advance.
Rai

**Comment by samboy** [comment, marked as answer]
That’s an excellent feature request! I will see when I get time to implement it.

— Sam
